src/components/NovelCover.tsx

```diff
@@ -35,23 +36,37 @@ interface DownloadBadgeProps {
   theme: ThemeColors;
 }
 
-function NovelCover<TNovel extends NovelItem | NovelInfo | LibraryNovelInfo>({
-  item,
-  onPress,
-  libraryStatus,
-  theme,
-  isSelected,
-  onLongPress,
-  selectedNovelIds,
-}: {
+type coverItemLibrary =
+  | LibraryNovelInfo & {
+      completeRow?: number;
+    };
+
+type coverItemPlugin =
+  | NovelItem & {
+      completeRow?: number;
+    };
```

Comment:
```suggestion
type CoverItemLibrary =
  | LibraryNovelInfo & {
      completeRow?: number;
    };

type CoverItemPlugin =
  | NovelItem & {
      completeRow?: number;
    };
```
just for code conventions